`EpwPrepWorkChain`: new sub-process `epw_bands`

ymzhang0 opened the pull request:
I added the new `EpwBaseWorkChain` as a sub-process with label `epw-bands` in the `EpwPrepWorkChain`.

This sub-process shares the same structure as `epw` sub-processes.

Problem in the protocol files is also fixed. All the `metadata` namespaces are removed so that `option` is exposed at the top level.

mbercx reviewed (commented):
Thanks @ymzhang0! Sorry for the slow review. I just have a few small comments, but most changes look great. 👍

ymzhang0 committed:
New feature

I added an additional `EpwBaseWorkChain` subprocess in EpwPrepWorkChain
that do a subsequent `epw.x` interpolation of electron and phonon bands
after bloch to wannier transformation.

The related part in protocol yaml file is also modified accordingly.

mbercx reviewed (approved):
Thanks @ymzhang0! Also for going through the effort of separating the changes, much appreciated. This one looks good to go for me. 🚀 

As with the other PR, feel free to "Squash and Merge", see my comment [here](https://github.com/aiidaplugins/aiida-epw/pull/15#issuecomment-3615414848).

ymzhang0 committed:
Merge branch 'main' into feat/bandint
